national_rail/extract.py

```diff
@@ -0,0 +1,27 @@
+"""Gets the information from the national rail api"""
+
+from os import environ as ENV
+
+from requests import get
+from dotenv import load_dotenv
+
+
+def get_data_from_api() -> None:
+    """Gets the data from the national rail api and saves it to a file"""
+    load_dotenv()
+
+    url = "https://api1.raildata.org.uk/1010-knowlegebase-incidents-xml-feed1_0/incidents.xml"
+
+    headers = {
+        "x-apikey": ENV["APIKEY"],
+        "User-Agent": ""
```

Comment:
Why is this User agent empty? Is it needed?

Reply:
Yes so that the request can work. It is probably used as a tag